File: dotnet-weblog/Models/Comment.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace myapp.Models {
    public class Comment {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Body { get; set; }

        public int post_id { get; set; }
        public Post Post { get; set; }
    }
}

File: rails-weblog/app/models/comment.rb
class Comment < ApplicationRecord
  belongs_to :post

  def self.seed
    Post.seed
    if Comment.count > 0
      return
    end
    Post.find_each do |post|
      10.times do
        post.comments.create!(title: Faker::Lorem.sentence, body: Faker::Lorem.paragraphs(number: 2).join("\n"))
      end
    end
  end
end
